README.md

```diff
@@ -0,0 +1,74 @@
+[![PyPI](https://badge.fury.io/py/mssql-cli.svg)](https://pypi.python.org/pypi/mssql-cli)
+[![Python 2.7|3.4|3.5|3.6|3.7|3.8](https://img.shields.io/badge/python-2.7&nbsp;|&nbsp;3.4&nbsp;|&nbsp;3.5&nbsp;|&nbsp;3.6&nbsp;|&nbsp;3.7&nbsp;|&nbsp;3.8-blue.svg)](https://github.com/dbcli/mssql-cli)
+
+# mssql-cli
+[**mssql-cli**](https://github.com/dbcli/mssql-cli) is an interactive command line query tool for SQL Server. This open source tool works cross-platform and proud to be a part of the [dbcli](github.com/dbcli) community. 
+
+![mssql-cli Autocomplete](https://github.com/dbcli/mssql-cli/raw/master/screenshots/mssql-cli-autocomplete.gif)
+
+mssql-cli supports a rich interactive command line experience, with features such as:
+- **Auto-completion**: fewer keystrokes needed to complete complicated queries.
+- **Syntax highlighting**: highlights T-SQL keywords.
+- **Query history**: easily complete an auto-suggested query that was previously executed.
+- **Configuration file support**: customize the mssql-cli experience for your needs.
+- **Multi-line queries**: execute multiple queries at once using the multi-line edit mode.
+- **Non-interactive support**: execute a query without jumping into the interactive experience.
+
+## Quick Start
+Read the section below to quickly get started with mssql-cli. Consult the [usage guide](https://github.com/dbcli/mssql-cli/tree/master/doc/usage_guide.md) for a deeper dive into mssql-cli features.
+
+### Install mssql-cli
+Platform-specific installation instructions are below:
+| [Windows](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/windows.md#windows-installation) | [macOS](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/macos.md#macos-installation) | [Linux](https://github.com/dbcli/mssql-cli/blob/master/doc/installation/linux.md) |
+| - | - | - |
+
+Visit the [installation reference guide](https://github.com/dbcli/mssql-cli/tree/master/doc/installation) to view all supported releases and downloads.
+
+#### Install with Linux Package Manager
+> A new stable version of mssql-cli is coming March 2020 for `apt-get` and `yum`.
+
+Follow the [Linux installation instructions]('https://github.com/dbcli/mssql-cli/blob/master/doc/installation/linux.md') to install mssql-cli using `apt-get`, `yum`, and other Linux package managers.
+
+#### Install with pip
+```
+pip install mssql-cli
```

Comment:
Let's use: `python -m pip install mssql-cli` and remove all other references to running pip directly